docs/guides/retry-configuration.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: "azapi"
+page_title: "Feature: Retry Configuration"
+description: |-
+  This guide will describe how to configure the retry behavior in the AzAPI provider. The retry configuration allows you to control how the provider handles failed requests and if it should retry them.
+---
+
+## Why retry?
+
+Sometimes, when managing cloud infrastructure, requests to the cloud provider may fail due to transient issues such as network problems, timeouts, eventual consistency, or rate limiting. In these cases, it can be beneficial to retry the request a few times before giving up. The AzAPI provider allows you to configure the retry behavior to suit your needs.
+
+## Retry configuration
+
+There are two types of retry configurations available in the AzAPI provider:
+
+1. Provider retry configuration
+2. Resource-specific retry configuration
+
+### Provider retry configuration
+
+The provider retry configuration is a global configuration that applies to all resources managed by the provider. You can configure the provider retry behavior by setting the following provider values:
+
+- `go_sdk_maximum_retry_attempts`
```

Comment:
The `go_sdk` prefix seems like an unnecessary / confusing implementation detail to me. As a user I might not even know that providers are implemented in go. I think it would be better to use just `maximum_retry_attempts`.

Reply:
It was to differentiate it from the resource specific retry. But I agree a better name is probably available. Will have a think!

Reply:
renamed... thoughts?